src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/mappers/ChildrenInfoMapper.java

```diff
@@ -46,10 +47,12 @@ private static ChildInfo mapChild(int index, List<OcrDataField> ocrDataFields) {
             dob = transformFormDateIntoCcdDate("childInfo" + index + ".dateOfBirth", childsDob);
         }
 
+        String gender = getValueOrEmptyString(index, ocrDataFields, GENDER);
+
         return ChildInfo.builder()
             .name(getValueOrEmptyString(index, ocrDataFields, NAME_OF_CHILD))
             .dateOfBirth(dob)
-            .gender(getValueFromOcrDataFields(GENDER + index, ocrDataFields).orElse("notGiven"))
+            .gender(StringUtils.isNotBlank(gender) ? gender : "notGiven")
```

Comment:
Rather than use "notGiven" you can make use of the enum Gender - `NOT_GIVEN.getValue()` I think.